Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -494,3 +537,31 @@ function Get-ProxyUri
 
     return $proxyUri
 }
+
+function ConvertTo-Pfx {
+    param(
+        [String][Parameter(Mandatory = $true)] $pemFileContent
+    )
+
+    if ($ENV:Agent_TempDirectory) {
+        $pemFilePath = "$ENV:Agent_TempDirectory\clientcertificate.pem"
+        $pfxFilePath = "$ENV:Agent_TempDirectory\clientcertificate.pfx"
+    }
+    else {
+        $pemFilePath = "$ENV:System_DefaultWorkingDirectory\clientcertificate.pem"
+        $pfxFilePath = "$ENV:System_DefaultWorkingDirectory\clientcertificate.pfx"    
+    }
+
+    # save the PEM certificate to a PEM file
+    Set-Content -Path $pemFilePath -Value $pemFileContent
+
+    # using openssl to convert the PEM file to a PFX file
+    $pfxFilePassword = [System.Guid]::NewGuid().ToString()
+    
+    $openSSLExePath = "$PSScriptRoot\openssl\openssl.exe"
+    $openSSLArgs = "pkcs12 -export -in $pemFilePath -out $pfxFilePath -password pass:$pfxFilePassword"
```

Comment:
You are passing secret in command line. Can you avoid this. Can you check with @arjgupta who fixed similar issues for other tasks.